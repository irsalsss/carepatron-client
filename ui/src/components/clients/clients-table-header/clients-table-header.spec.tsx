import { render } from "@/utils/test/react-testing-setup";
import ClientsTableHeader from "./clients-table-header";

describe("ClientsTableHeader", () => {
  it("should render client table header successfully", () => {
    const { baseElement } = render(<ClientsTableHeader />);

    expect(baseElement).toMatchSnapshot();
    expect(baseElement).toBeTruthy();
  });
});
