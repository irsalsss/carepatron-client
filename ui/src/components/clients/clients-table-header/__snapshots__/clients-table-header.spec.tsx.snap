// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClientsTableHeader should render client table header successfully 1`] = `
<body>
  <div>
    <div
      class="grid grid-cols-12 gap-6 py-6 px-10 border-b-[2px] border-neutral-100"
    >
      <div
        class="col-span-4"
      >
        <span
          class="font-bold"
        >
          Name
        </span>
      </div>
      <div
        class="col-span-4"
      >
        <span
          class="font-bold"
        >
          Phone
        </span>
      </div>
      <div
        class="col-span-4"
      >
        <span
          class="font-bold"
        >
          Email
        </span>
      </div>
    </div>
  </div>
</body>
`;
